src/utils/auth.js

```diff
@@ -13,8 +13,14 @@ function createAuthProvider(firebase, providerName, scopes) {
   // TODO: Verify scopes are valid before adding
   // TODO: Validate parameter inputs
 
-  if (providerName.toLowerCase() === 'microsoft.com') {
-    const provider = new firebase.auth.OAuthProvider('microsoft.com')
+  const lowerCaseProviderName = providerName.toLowerCase()
+
+  if (
+    lowerCaseProviderName === 'microsoft.com' ||
```

Comment:
Most likely going to update this to be an array instead of multiple if conditions since that will also make it easy to add to the list:
```js
['microsoft.com', 'apple.com', 'yahoo.com'].includes(lowerCaseProviderName)
```